versioned_docs/version-22.04/releases/centreon-core.md

```diff
@@ -17,6 +17,23 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 22.04.12
+
+Release date: `March AA, 2023`
+
+#### Bug fixes
+
+- [CLAPI] Improved authentication by not checking password expiration policy for excluded users.
+- [Configuration] Fixed an issue in the recurrent downtimes form that caused service relations to be lost.
+- [Configuration] Fixed global configuration export button in banner with OIDC authentication.
+- [Core] Fixed blank page after login.
+- [UI] Fixed an issue where a popup does not appear in legacy page graphs.
```

Comment:
```suggestion
- [UI] Fixed an issue where a popup was not displayed in the graphs legacy page.
```

Reply:
the issue is found in graphs of a certain legacy page, not a page where we only have graphs